README.md

```diff
@@ -1,4 +1,4 @@
-# Mor<a name="readme-top">School-library</a>
+# <a name="readme-top">School-library</a>
 
 # 📗 Table of Contents
 
```

Comment:
- [ ] To make your README documentation be more professional and more descriptive, kindly make sure you update both `## Show your support` and `## Getting Started` sections with appropriate information which can help a non technical user of how to use your project. I suggest you **[refer this template example](https://gist.github.com/tresorsawasawa/0eba0570695aef068a7837c349f94737)** to help you how to fill the  `## Getting Started` section

- [ ] I also suggest you improve the content of the `Getting Started` section to be more meaningful. Instead of the current content you put somethig like this below:
```

:telescope:  Future Features
- Add UI
- Etc
```